components/editor/CodeBlock.tsx

```diff
@@ -0,0 +1,583 @@
+/* eslint-disable unused-imports/no-unused-vars */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-useless-escape */
```

Comment:
Kindly remove `eslint-disable` comments like these

Reply:
Then I will have to remove the `title` variable from the   function arguments and modify regex in the code? Should I go ahead with this?